fixtures/eurolite/led-b-40.json

```
@@ -0,0 +1,425 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED B-40",
+  "categories": ["Laser", "Flower", "Effect", "Color Changer"],
+  "meta": {
+    "authors": ["Charlélie DESMET", "Flo Edelmann"],
```

```suggestion
    "authors": ["Charlélie Desmet", "Flo Edelmann"],
```

Author names should be lowercase with (optionally) an uppercase first letter. This should especially apply to personal names like this one.